lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -106,11 +106,11 @@ def setModel(self, model):
             TaurusLabel.setModel(self, None)
             self.setText(devName)
 
-    _BCK_COMPAT_TAGS = {'<attr_name>': '{attr.name}',
-                        '<attr_fullname>': '{attr.fullname}',
-                        '<dev_alias>': '{dev.name}',
-                        '<dev_name>': '{dev_name}',
-                        '<dev_fullname>': '{dev.fullname}',
+    _BCK_COMPAT_TAGS = {'<attr_name>': 'name',
```

Comment:
I think that `TaurusValue.resetLabelConfig` needs also be changed for coherence with this

Reply:
Done